dev/io.openliberty.checkpoint_fat_persistence/fat/src/io/openliberty/checkpoint/fat/FATSuite.java

```diff
@@ -22,8 +22,9 @@
 @RunWith(Suite.class)
 @SuiteClasses({
                 AlwaysPassesTest.class,
-                DB2Test.class,
-                JPATest.class
+                DB2Test.class
+// Comment out JPA pending upcoming changes to support JPA with checkpoint
+// JPATest.class
```

Comment:
Why not call setAssertNoAppRestartOnRestore(false) for `JPATest`?

Reply:
The intended use of that setting is to signal that the FAT is exhibiting passing behavior even when the app restarts after restore. I think it's clearer to comment out the test since it's not working ATM.